### PR TITLE
[WOE] Implement Virtue of Courage

### PR DESCRIPTION
**Pull request by Susucre**

### CODE REVIEW

**Comment by Susucre** [comment]
[[Virtue of Courage]]

File: Mage.Sets/src/mage/cards/v/VirtueOfCourage.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import mage.target.common.TargetAnyTarget;
+
+import java.util.UUID;
+
+/**
+ * @author Susucr
+ */
+public final class VirtueOfCourage extends AdventureCard {
+
+    private static final FilterStackObject filter
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/v/VirtueOfCourage.java:24]: what's up with this? seems irrelevant

Comment (Susucre) [reply Mage.Sets/src/mage/cards/v/VirtueOfCourage.java:24]: Yup, looks like I did start by copying another card, and forgot to cleanup that filter.

**Review by xenohedron — APPROVED** [review]
